zscript/StupidAchievements.zs

```diff
@@ -473,6 +497,7 @@ class sa_Task abstract
   protected int       mNLines;
   protected Font      mFont;
   protected TextureID mTexture;
+  protected TextureID mIcon;
```

Comment:
mIcon is not initialized, no icon shows on on-screen achievement.

Reply:
Can I ask how to fix it?